docs/user/recipes/recipes_list.md

```diff
@@ -201,6 +201,12 @@ The list of available quacc recipes is shown below. The "Req'd Extras" column sp
 | Espresso DOS Flow        | `#!Python @flow` | [quacc.recipes.espresso.dos.dos_flow][]             |              |
 | Espresso Projwfc         | `#!Python @job`  | [quacc.recipes.espresso.dos.projwfc_job][]          |              |
 | Espresso Projwfc Flow    | `#!Python @flow` | [quacc.recipes.espresso.dos.projwfc_flow][]         |              |
+| Espresso Bands Job       | `#!Python @job`  | [quacc.recipes.espresso.bands.bands_job][]          |              |
+| Espresso Bands           | `#!Python @job`  | [quacc.recipes.espresso.bands.bands][]              |              |
+| Espresso Bands PP        | `#!Python @job`  | [quacc.recipes.espresso.bands.bands_pp][]           |              |
+| Espresso Fermi Surface   | `#!Python @job`  | [quacc.recipes.espresso.bands.fermi_surface][]      |              |
```

Comment:
The naming convention in `quacc` is that all jobs end in `_job` (and all flows with `_flow`). Here, can you work on renaming some of these accordingly? You will have to be a bit creative with the two different bands jobs. 

The naming helps users because they don't have to look at the source code to know what type of function it is.